### PR TITLE
Fix suffix for bus messages

### PR DESCRIPTION
**Pull request by simao**
Add more logging

### CODE REVIEW

File: common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala [hunk 1]
```diff
@@ -42,11 +42,11 @@ object KinesisClient {
       awsSecretKey <- awsConfig.readString("secretAccessKey")
     } yield new StaticCredentialsProvider(new BasicAWSCredentials(awsAccessKey, awsSecretKey))
 
-  private[this] def getStreamNameSuffix(streamNameRoot: String, config: Config): ConfigException Xor String =
+  private[this] def streamNameWithSuffix(config: Config): ConfigException Xor (String => String) =
     for {
-      awsConfig <- config.configAt("aws")
-      streamNameSuffix <- config.readString("streamNameSuffix")
-    } yield streamNameRoot + "-" + streamNameSuffix
+      awsConfig <- config.configAt("messaging.kinesis")
```
Comment (CalumMcCall) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: I still get config key not found exceptions when testing this fix locally. Have you tested it?

Comment (simao) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: yes, you also need the next like `config => awsConfig`

Comment (CalumMcCall) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: Ah yes, I missed that piece!

Comment (CalumMcCall) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: Can we rename this `awsConfig` variable?

**Comment by CalumMcCall** [comment]
Once the awsConfig variable is renamed, it's good to merge from my end.

11/11

File: common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala (hunk 1 above)
Comment (simao) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: to what?

Comment (CalumMcCall) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala]: `cfg` like the rest of the file. Or `kinesisConfig`, something like that.

**Pull request merged**